ee/clickhouse/sql/dead_letter_queue.py

```diff
@@ -40,7 +40,7 @@
     cluster=CLICKHOUSE_CLUSTER,
     extra_fields=KAFKA_COLUMNS,
     engine=table_engine(DEAD_LETTER_QUEUE_TABLE, "_timestamp", REPLACING_MERGE_TREE),
-    ttl_period=ttl_period("_timestamp", 12),  # 12 weeks = 3 months
+    ttl_period=ttl_period("_timestamp", 4),  # 4 weeks
```

Comment:
Updated TTL to 4 weeks based on your feedback about storage @guidoiaquinti 

The topic in Heroku Kafka has 3 day retention (we can change that), so we wouldn't have a lot of duplication, but realistically don't need to keep records in CH for that long.

If we run into a problem like DB going down, then we can hit Kafka directly and produce to the ingestion topic again off a consumer for the DLQ as soon as things are fine.

But if we have a deeper problem that requires some analysis and data parsing then we just default to CH and have more time to come up with a plan.